src/Uno.UI.RuntimeTests/Tests/Microsoft_UI_Xaml_Controls/Given_TreeView.cs

```diff
@@ -94,6 +95,104 @@ public async Task When_TreeViewItem_Dragged_Near_the_Edge()
 		}
 #endif
 
+		[TestMethod]
+#if __ANDROID__ || __IOS__
+		[Ignore("Fails")]
```

Comment:
any explanation here?

Reply:
The changes in this PR fix this only for the managed VirtualizingPanels. I'm not sure what the situation with mobile targets is, but a corresponding fix will need to be added there as well. I'm not very familiar with the virtualization implementation on iOS and Android (and also I can't test locally), so I was hoping to get some help there.

Reply:
you could add this in a comment